lib/src/test/java/uk/gov/di/ipv/core/library/service/IpvSessionServiceTest.java

```diff
@@ -59,6 +59,23 @@ void shouldReturnSessionItem() {
         assertEquals(ipvSessionItem.getCreationDateTime(), result.getCreationDateTime());
     }
 
+    @Test
+    void shouldReturnSessionItemByCriOAuthSessionId() {
```

Comment:
I think, we can add mor one test for null scenario

Reply:
We don't have tests like this for `getIpvSessionByAuthorizationCode` or `getIpvSessionByAccessToken` which are similar methods - I think this is fine as is